cfme/control/explorer/alerts.py

```diff
@@ -304,6 +306,30 @@ def create(self, description, severity=None, active=None, based_on=None, evaluat
         view.flash.assert_success_message('Alert "{}" was added'.format(alert.description))
         return alert
 
+    def all(self):
+        # get alerts via reading 'All Alerts table'
+        view  = navigate_to(self.appliance.collections.alerts,'All')
+        try:
+            alert_table = view.table
+            alerts = [
+                self.instantiate(
+                      alert.get('Description'),
+                      active=alert.get('Active'),
+                      based_on=alert.get('Based On'),
+                      evaluate=alert.get('What is evaluated'),
+                      emails=alert.get('Email'),
+                      snmp_trap=alert.get('SNMP'),
+                      timeline_event=alert.get('Event on Timeline'),
+                      mgmt_event=alert.get('Management Event Raised')
+                )
+                for alert in alert_table.read()
+            ]
+        except AttributeError as e:
+            logger.error('Table not found in Alerts All view')
+            logger.error(e)
```

Comment:
**Required** I dont think you need to log the exception here. Was there a reason why you wanted to log it. 

This could be simplified to just:
```except AttributeError:
￼             logger.error('AlertCollection: Table not found in Alerts All view')```

Reply:
No reason specifically, just thought it may be good to log the full exception. Will be simplified in next iteration.